tests/SiteTreeContentReviewTest.php

```diff
@@ -334,4 +334,34 @@ public function testSiteConfigSettingsAreUsedAsDefaults()
 
         SS_Datetime::clear_mock_now();
     }
+    public function testSiteSettingsFieldsListBox()
+    {
+        $author = $this->objFromFixture("Member", "editor");
+
+        $this->logInAs($author);
+		
+        /** @var Page|SiteTreeContentReview $page */
+        $page = $this->objFromFixture("Page", "contact");
+
+        $fields = $page->getSettingsFields();
+		
+        $this->assertTrue($fields->dataFieldByName('OwnerUsers') instanceof ListboxField);
```

Comment:
You can use `$this->assertInstanceOf('ListboxField', $fields->da...`, same as line 365